pkg/jtracer/jtracer.go

```diff
@@ -84,11 +89,6 @@ func initHelper(
 	// span processor to aggregate spans before export.
 	bsp := sdktrace.NewBatchSpanProcessor(traceExporter)
 
-	res, err := otelResource(ctx, svc)
-	if err != nil {
-		return nil, err
```

Comment:
this was leaking exporter created above. Better to create resource first since it doesn't start any background work.